include/ht_debug_camera.h

```diff
@@ -0,0 +1,52 @@
+/**
+**    Hatchit Engine
+**    Copyright(c) 2015 Third-Degree
+**
+**    GNU Lesser General Public License
+**    This file may be used under the terms of the GNU Lesser
+**    General Public License version 3 as published by the Free
+**    Software Foundation and appearing in the file LICENSE.LGPLv3 included
+**    in the packaging of this file. Please review the following information
+**    to ensure the GNU Lesser General Public License requirements
+**    will be met: https://www.gnu.org/licenses/lgpl.html
+**
+**/
+
+#pragma once
+
+#include <ht_camera.h>
+#include <ht_math.h>
+#include <ht_time_singleton.h>
+#include <ht_input_singleton.h>
+
+namespace Hatchit {
+
+    namespace Game {
+
+        class HT_API DebugCamera
```

Comment:
Any reason we aren't using existing camera component?  Debug camera could just be a controllable game object with a camera component?